packages/utils/__tests__/path-utils.test.ts

```diff
@@ -0,0 +1,13 @@
+import { join } from "path";
+import { isLocalPath, findProjectRoot } from "../path-utils";
+
+describe("path-utils tests", () => {
+  it("isLocalPath Testing : __dirname", () => {
+    const answer = isLocalPath(join(__dirname, "../../"));
```

Comment:
You should test this against different formats:
- ./path
- path/path
- ..\path (win)

And also you should test the error cases